__tests__/index.js

```diff
@@ -5,19 +5,59 @@ import nock from 'nock'
 import {autocomplete, bulk, reverse, search} from '../index'
 
 const mockBulkResult = require('./mock-bulk-result.json')
+const mockBulkResultWithBadAddress = require('./mock-bulk-result-with-bad-address.json')
 const mockReverseResult = require('./mock-reverse-result.json')
 const mockSearchResult = require('./mock-search-result.json')
 const mockSuggestResult = require('./mock-suggest-result.json')
 
+function prepareNock (
+  type: 'autocomplete' | 'bulk' | 'search' | 'reverse',
+  snapshotUri = true,
+  response?: Object
+) {
+  if (type === 'bulk') {
```

Comment:
This should be done in a `beforeEach` inside the `bulk` scope: https://facebook.github.io/jest/docs/en/setup-teardown.html